lib/pbench/cli/server/shell.py

```diff
@@ -188,17 +188,13 @@ def main():
     if site.ENABLE_USER_SITE:
         find_the_unicorn(logger)
     try:
-        host = str(server_config.get("pbench-server", "bind_host"))
+        host = server_config.get("pbench-server", "bind_host")
         port = str(server_config.get("pbench-server", "bind_port"))
-        db_uri = str(server_config.get("database", "uri"))
+        db_uri = server_config.get("database", "uri")
```

Comment:
Huh.  You removed the `str()`'s from lines 191 and 193, but left the one on line 192.  Curious.  I realize that we want a string value...does `get()` produce a numeric result for things which are numeric?  That's kind of magical!